File: Examples/CSharp/Programming-Documents/Fields/UpdateDocFields.cs
using System;
using System.Collections;
using System.IO;
using Aspose.Words;
using Aspose.Words.Tables;
using Aspose.Words.Fields;
using Aspose.Words.Layout;

namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_with_Fields
{
    class UpdateDocFields
    {
        public static void Run()
        {
            //ExStart:UpdateDocFields
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithFields();

            Document doc = new Document(dataDir + "Rendering.doc");

            // This updates all fields in the document.
            doc.UpdateFields();
            dataDir = dataDir + "Rendering.UpdateFields_out_.pdf";
            doc.Save(dataDir);
            //ExEnd:UpdateDocFields
            Console.WriteLine("\nDocument fields updated successfully.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/VisualBasic/VisualBasic/Programming-Documents/Fields/UpdateDocFields.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Words
Imports Aspose.Words.Fields
Public Class UpdateDocFields
    Public Shared Sub Run()
        ' ExStart:UpdateDocFields
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithFields()

        Dim doc As New Document(dataDir & Convert.ToString("Rendering.doc"))

        ' This updates all fields in the document.
        doc.UpdateFields()
        dataDir = dataDir & Convert.ToString("Rendering.UpdateFields_out.pdf")
        doc.Save(dataDir)
        ' ExEnd:UpdateDocFields
        Console.WriteLine(Convert.ToString(vbLf & "Fields updated successfully." & vbLf & "File saved at ") & dataDir)
    End Sub
End Class
